### PR TITLE
Update manual benchmarks to make it easier to add simple models

### PR DESCRIPTION
**Pull request by jpienaar**
Make it easy to add simple models where we don't care about the input and can use TFLite and IREE's input generation.

Also fix typo in returned file name (missed last time as I had file with typo in it on device too).

### CODE REVIEW

File: build_tools/benchmarks/comparisons/simple_commands.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                     num_threads,
+                     num_runs,
+                     taskset=taskset)
+    self.driver = driver
+    self.args.append("--entry_function=main")
+    self.args.append('--function_input="%s"' % function_input)
+
+
+class SimpleCommandFactory(BenchmarkCommandFactory):
+  """Generates `BenchmarkCommand` objects specific to running series of simple models."""
```
Comment (GMNGeoffrey) [inline build_tools/benchmarks/comparisons/simple_commands.py:76]: Can you elaborate on what "simple" is here?

File: build_tools/benchmarks/comparisons/simple_commands.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+  def __init__(self, base_dir: str):
+    self._models = [
+        Model("mobilenet_v2_1.0_224", "1x224x224x3xf32", None, None),
+        Model("mobilenet_v2_224_1.0_uint8", "1x224x224x3xui8", "input",
+              "1,224,224,3"),
```
Comment (GMNGeoffrey) [inline build_tools/benchmarks/comparisons/simple_commands.py:82]: The init of a class seems like a surprising place to have this list. Can we instead make this get passed in?

**Review by GMNGeoffrey — APPROVED** [review]
typo in "typo"

inception

File: build_tools/benchmarks/comparisons/simple_commands.py (hunk 2 above)
Comment (jpienaar) [reply build_tools/benchmarks/comparisons/simple_commands.py]: Let me try that, I'll move the looping to run_benchmark effectively and just have this set.

**Pull request merged**